readme.md

```diff
@@ -188,6 +188,13 @@ You can pass an empty string to remove the suffix.
 
 For example, on macOS, the config file will be stored in the `~/Library/Preferences/foo-nodejs` directory, where `foo` is the `projectName`.
 
+#### disableDotNotation
```

Comment:
Double-negative option names are not optimal. Would be better with `dotNotation`.